index.html

```diff
@@ -55,97 +55,8 @@ <h2 class="secondtitle">I'm a software developer</h2>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-        <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-        </div>
+        <div class="card"></div>
```

Comment:
- [ ]  You should position popup contents referring to provided Figma design.  In your design, buttons are not well adjusted and the provided texts are very small, I guess it is better to add text length closer to the one provided in the Figma design.  This will make your popup look more professional. 
- [ ]  Your cross button should be in the right side, as it is provided in Figma design.  

Figma Design             | Your design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/43877689/175431869-4bff5efe-d0ac-42e9-a39f-a651cc1dc82b.png)  |  ![image](https://user-images.githubusercontent.com/43877689/175430895-fcb85d3f-7e6c-4ac8-844a-1660b6789ad5.png)